src/theme/variables.scss

```diff
@@ -38,7 +38,8 @@ $colors: (
   secondary:  #32db64,
   danger:     #f53d3d,
   light:      #f4f4f4,
-  dark:       #222
+  dark:       #222,
+  grey: rgba(0, 0, 0, 0.35)
```

Comment:
Choose one color definition

Reply:
The ionic build does not support 4/8 digit hex code to include transparency yet